tools/cas/cas.go

```diff
@@ -66,16 +66,25 @@ func main() {
 	if *blobDigest == "" {
 		log.Fatalf("Missing --digest")
 	}
-	d, err := digest.Parse(*blobDigest)
+	var ind *digest.ResourceName
+	var err error
+
+	if *blobType == "ActionResult" {
+		ind, err = digest.ParseActionCacheResourceName(*blobDigest)
```

Comment:
IIUC this means the `-digest` flag requires a `blobs/ac` prefix now, or `blobs/` if you're fetching from CAS, which feels a bit more cumbersome

Would be nicer imo to build the resource name string based on the existing digest / instance name flags (using /ac/ only if blobtype is ActionResult), so that we could remove the backwards compatibility code

Reply:
Fixed this.